src/wasm/wasm-stack.cpp

```diff
@@ -3213,12 +3213,9 @@ void StackIRToBinaryWriter::write() {
       case StackInst::LoopBegin:
       case StackInst::TryTableBegin: {
         if (sourceMap) {
-          parent.writeDebugLocation(inst->origin, func);
+          parent.writeSourceMapLocation(inst->origin, func);
         }
         writer.visit(inst->origin);
-        if (sourceMap) {
-          parent.writeDebugLocationEnd(inst->origin, func);
-        }
```

Comment:
This was just unneeded - that code did nothing when source maps were enabled. Source maps just do not track the ends of things.